site/documentation/content/api/command-and-control-pubsub/index.md

```diff
@@ -121,18 +145,29 @@ command message.
 |:---------------------------------------------|:---------:|:----------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | *correlation-id*                             |    yes    | *string*  | The identifier used to correlate a response message to the original request. It is used as the *correlation-id* attribute in the response.                                                                                                                                                                                                                                                            |
 | *device_id*                                  |    yes    | *string*  | The identifier of the device that the command is targeted at.                                                                                                                                                                                                                                                                                                                                         |
-| *response-required*                          |    yes    | *boolean* | MUST be set with a value of `true`, meaning that the device is required to send a response for the command.                                                                                                                                                                                                                                                                                           |
+| *response-required*                          |    no     | *boolean* | Either *response-required* or *ack-required* MUST exclusively be set with a value of `true` (XOR). *response-required* set to `true` means that the device is required to send a response for the command.                                                                                                                                                                                            |
+| *ack-required*                               |    no     | *boolean* | Either *response-required* or *ack-required* MUST exclusively be set with a value of `true` (XOR). *ack-required* set to `true` means that the protocol adapter will try to send an acknowledgement response for the command on behalf of the device in case the command was successfully received by the device. This mechanism has some limitations which are described in the info field below.    |
```

Comment:
```suggestion
| *ack-required*                               |    no     | *boolean* | `true` if the protocol adapter should send an acknowledgement response for the command on behalf of the device in case the command was successfully received by the device. This mechanism has some limitations which are described in the info field below. *response-required* and *ack-required* MUST NOT both be set to `true` at the same time. |
```